copter/source/docs/reference-frames-hexsoon-td650.rst

```diff
@@ -22,31 +23,73 @@ Box Contents
 - Landing Gear
 - Power Distribution Board
 
-Additional Required Parts
--------------------------
+Additional Parts
+----------------
 
 - Compatible with the :ref:`Cube autopilot <common-thecube-overview>` with direct mounting screws for the standard carrier board
 - 8000mAh to 12000mAh 6S battery
 - RC transmitter with at least 6 channels
+- (Optional) four `5cm (2inch) C-clamps <https://www.amazon.com/C-Clamp-Industrial-Strength-Woodworking-Building/dp/B0BLC44XHT>`__ to ease assembly
+- (Optional) `solder sleeves <https://www.amazon.com/TICONN-Connectors-Waterproof-Insulated-Electrical/dp/B07GDDKJ1D>`__ to shorten ESC wires
+- (Optional) 2cm x 50cm velcro battery straps to replace very short straps provided
 
 Connection and Setup
 --------------------
 
-.. image:: ../images/refrence-frames-hexsoon-edu650-assembly.jpg
-    :target: ../_images/refrence-frames-hexsoon-edu650-assembly.jpg
+.. image:: ../images/reference-frames-hexsoon-edu650-assembly.jpg
+    :target: ../_images/reference-frames-hexsoon-edu650-assembly.jpg
 
 Connect the four ESC wires to the back of the autopilot as shown in the :ref:`QuadX configuration <connect-escs-and-motors>`
 
 Parameter file: `hexsoon-td650.param <https://github.com/ArduPilot/ardupilot/blob/master/Tools/Frame_params/hexsoon-td650.param>`__
 
 This parameter file can also be loaded using the Mission Planner's Config/Tuning >> Full Parameter Tree page by selecting "hexsoon-td650" from the drop down on the middle right and then push the "Load Presaved" button.
 
+LED connection and configuration
+--------------------------------
+
+The LED cable from each arm should be connected to any of PDB's six LED ports shown in green below (the order does not matter).
```

Comment:
```suggestion
The LED cable from each arm should be connected to any of PDB's six LED ports shown in green below (the order does not matter, but the LED ports must be filled lowest-to-highest).
```

Reply:
Txs, I've changed the wording to, "The LED cable from each arm should be connected to the PDB's LED ports labelled "1" to "4" and shown in green below (the order does not matter)."